src/libs/actions/PersonalDetails.js

```diff
@@ -252,8 +252,8 @@ function fetchCurrencyPreferences() {
         .then((currencyList) => {
             Onyx.merge(ONYXKEYS.MY_PERSONAL_DETAILS,
                 {
-                    preferredCurrencyCode: currency,
-                    preferredCurrencySymbol: currencyList[currency].symbol,
+                    localCurrencyCode: currency,
+                    localCurrencySymbol: currencyList[currency].symbol,
```

Comment:
Another place where probably we can just store the code and not both code and symbol as their relationship doesn't ever change.